Sources/Reactive.swift

```diff
@@ -11,6 +11,11 @@ extension ReactiveExtensionsProvider {
 	public var reactive: Reactive<Self> {
 		return Reactive(self)
 	}
+
+	/// A proxy which hosts static reactive extensions for the type of `self`.
+	public static var reactive: Reactive<Self>.Type {
+		return Reactive<Self>.self
+	}
```

Comment:
Huh. I didn't think you could have a static and instance members with the same name. (Did that change in Swift at some point?)

Nicely done!